code/swb_player/CameraMovement.cs

```diff
@@ -1,92 +1,126 @@
-using SWB.Shared;
+﻿using SWB.Shared;
 
 namespace SWB.Player;
 
-[Group( "SWB" )]
-[Title( "CameraMovement" )]
+[Group("SWB")]
+[Title("CameraMovement")]
 public class CameraMovement : Component
 {
-	[Property] public float Distance { get; set; } = 0f;
```

Comment:
You still using spaces for idents instead of tabs, please fix. I can also reformat all files you pushed if you can't figure it out  just let me know.